index.md

```diff
@@ -1324,6 +1324,28 @@ One thing for bot authors to note is that the `NOTICE` message may be interprete
 
 These messages are not required for a server implementation to work, but SHOULD be implemented. If a command is not implemented, it MUST return the [`ERR_UNKNOWNCOMMAND`](#errunknowncommand-421) numeric.
 
+### AWAY message
+
+         Command: AWAY
+      Parameters: [ <text> ]
+
+With the `AWAY` command, clients can indicate their user is away, with a message.
+The `AWAY` command is used either with one parameter, to set an `AWAY` message, or with no parameters, to remove the AWAY message.
+
+The server acknowledges the change in away status with `RPL_UNAWAY` or `RPL_NOWAWAY`.
+The server MAY forward the `AWAY` message to other clients, according to the [IRCv3 `away-notify` Extension](https://ircv3.net/specs/extensions/away-notify.html).
+
+This away status or message SHOULD be sent by servers to clients when relevant, including:
+
+1. [`RPL_AWAY`](#rplaway-301), with the away message, as automatic reply string for any PRIVMSG commands directed at the away user (not to a channel they are on).
+2. [`RPL_AWAY`](#rplaway-301), with the away message, in replies to [`WHOIS`](#whois-message) messages.
+3. [`RPL_USERHOST`](#rpluserhost-302), as the `+` or `-` character
```

Comment:
```suggestion
The `AWAY` command lets clients indicate that their user is away.
If this command is sent with a parameter (the 'away message') then the user is set to be away. If this command is sent with no parameters, the user is no longer away.

The server acknowledges the change in away status by returning the `RPL_NOWAWAY` and `RPL_UNAWAY` numerics.
If the [IRCv3 `away-notify` capability](https://ircv3.net/specs/extensions/away-notify.html) is used, the server MAY also send clients `AWAY` messages to tell them how the away status of other users has changed.

Servers SHOULD notify clients when a user they're interacting with is away when relevant, including sending these numerics:

1. [`RPL_AWAY`](#rplaway-301), with the away message, when a PRIVMSG command is directed at the away user (not to a channel they are on).
2. [`RPL_AWAY`](#rplaway-301), with the away message, in replies to [`WHOIS`](#whois-message) messages.
3. In the [`RPL_USERHOST`](#rpluserhost-302) numeric, as the `+` or `-` character.
```

Reply:
"If the [IRCv3 `away-notify` capability](https://ircv3.net/specs/extensions/away-notify.html) is used"

I find this misleading, because it seems to mean that it's sent of the capability is used by the client sending the AWAY.

Reply:
Good point, reworded that sentence to make it more clear who requests/etc the cap.